examples/imitation_learning_openbot/generate_episodes.py

```diff
@@ -12,6 +12,38 @@
 
 from utils import plot_paths
 
+# import open_level functionality from common folder
+common_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), ".."))
+import sys
+sys.path.append(common_dir)
+from common.instance_utils import open_level
+
+
+class NavMesh:
```

Comment:
Move `NavMesh` to `common` because you're referring to it in multiple examples. And if you were only referring to it in this one example, a better place to put it would be in `utils.py`, not in `generate_episodes.py`.

Reply:
Makes sense. Moved it to `common`.